panther_driver/src/panther_hardware.py

```diff
@@ -93,6 +100,26 @@ def _setup_gpio() -> None:
         GPIO.setup(DRIVER_EN, GPIO.OUT, initial=0)
         GPIO.setup(E_STOP_RESET, GPIO.IN)  # USED AS I/O
 
+    @staticmethod
+    def _shutdown_host() -> None:
```

Comment:
This could alternatively be done by a class eg. howewer we wont use this class for antything but sending shutdown reqest so I opted against it. 
```
class HostSshClient:
    def __init__(self) -> None:
        # EXAMPLE: https://www.linode.com/docs/guides/use-paramiko-python-to-ssh-into-a-server/

        self.host = "10.15.20.2"
        self.username = "husarion"

        self.pkey = paramiko.RSAKey.from_private_key_file("/root/.ssh/id_rsa")
        self.client = paramiko.SSHClient()
        policy = paramiko.AutoAddPolicy()
        self.client.set_missing_host_key_policy(policy)

    def send_command(self, command: str) -> None:
        self.client.connect(self.host, username=self.username, pkey=self.pkey)
        self.client.exec_command("command")
        self.client.close()
```